components/Sidebar/sidebar.module.css

```diff
@@ -0,0 +1,155 @@
+.wrapper {
+  font: 700 16px 'DM Sans', sans-serif;
+  margin-bottom: 25px;
```

Comment:
Avoid using px.
Read here https://css-tricks.com/is-it-better-to-use-ems-rems-than-px-for-font-size/